docs/run-command.md

```diff
@@ -26,12 +43,24 @@ nodes:
     aks-agentpool-12345678-vmss000002:
         instance-id: "2"
         [...]
+```
+
+Start using one of above node e.g `aks-agentpool-12345678-vmss000000` we call it `$myNode` here:
+
+```bash
+kubectl aks config use-node $myNode
+```
+
+Execute the run-command, and it will be automatically executed in `aks-agentpool-12345678-vmss000000`: 
 
-# Start using one of those nodes
-$ kubectl aks config use-node aks-agentpool-12345678-vmss000000
+```bash
+kubectl aks run-command "ip route"
+```
 
-# Execute the run-command, and it will be automatically executed in aks-agentpool-12345678-vmss000000
-$ kubectl aks run-command "ip route"
+The output should be similar to this:
+
+<!--expected_similarity=0.8-->
+```
```

Comment:
I'm wondering if we should use another command that provides a more deterministic output so that we can use `<!--expected_similarity=1-->` or some regex to match an expected string.

Of course, this is something we can do later, not for this PR.